example/package.json

```diff
@@ -4,20 +4,17 @@
   "private": true,
   "scripts": {
     "start": "node node_modules/react-native/local-cli/cli.js start",
-    "test": "jest",
-    "lint": "eslint ."
+    "test": "jest"
   },
   "dependencies": {
     "react": "16.8.3",
-    "react-native": "0.59.3",
+    "react-native": "0.59.5",
     "react-native-device-info": "github:react-native-community/react-native-device-info"
   },
   "devDependencies": {
     "@babel/core": "7.4.0",
     "@babel/runtime": "7.4.2",
-    "@react-native-community/eslint-config": "0.0.3",
```

Comment:
The eslint items are things we are in the process of implementing per react-native-community standards - can you leave those in? Also, just because everything moves quickly - if you're in there RN0.59.8 is out now, but it's just some minor version bumps - easy to include: https://github.com/react-native-community/rn-diff-purge/compare/release/0.59.5..release/0.59.8

That said, could you use the refresh-example script to make the changes, to insure that it works and tracks the example refreshes correctly? RN0.60 is coming shortly and I want to make sure the example refresh is just a single command line away...